frontend/src/components/regions/CardDetails/Review.jsx

```diff
@@ -5,17 +5,25 @@ import CardHeader from "@material-ui/core/CardHeader";
 import CardContent from "@material-ui/core/CardContent";
 import IconButton from "@material-ui/core/IconButton";
 import Typography from "@material-ui/core/Typography";
-import Markdown from "react-markdown";
 import ReviewStatus from "../../widgets/ReviewStatus";
 import ReviewValidationIcons from "../../widgets/ReviewValidationIcons";
-// import ShowMoreText from "react-show-more-text";
-// import ExpandLess from "@material-ui/icons/ExpandLess";
-// import ExpandMore from "@material-ui/icons/ExpandMore";
+import { trimReviewComments } from "./utils";
+import Button from "@material-ui/core/Button";
+import ReviewPopUp from "./ReviewPopUp";
+import useOpenAndCloseReviewPopUp from "./hooks/openAndCloseReviewPopUp";
 
 const useStyles = makeStyles((theme) => ({
   root: {
     maxWidth: "100%",
   },
+  iconAlignment: {
+    position: "absolute",
+    right: "18px",
+    backgroundColor: "white",
+    "&:hover": {
```

Comment:
What is the effect of the hover, if the background is already white?

Do you really need to have it?

Reply:
Yes I do have to use it, otherwise, when I hover over "read more ...", it will be grayed out